extensions/interactions/interaction_specs.json

```diff
@@ -0,0 +1,874 @@
+{
```

Comment:
Shouldn't there be a corresponding deletion of these values from the place where it was written before? The values which were used earlier to populate the GLOBALS field.

Reply:
These are in form of the Python files in `extensions/interactions` and are still used in some places in backend. @Jamesjay4199 Coudl you file an issue to investigate this?

Reply:
The python files are used in the backend for some other purposes. That's why the values were not removed there.